src/ProgramRepresentation/ProgramPoint.cpp

```diff
@@ -14,24 +14,77 @@ ProgramPoint::ProgramPoint(std::string pointName, ProgramPoint *programPoint) {
 
 void ProgramPoint::addAlias(ProgramVariable receiving,
                             ProgramVariable receiver) {
+  std::string addedAliasRawName;
```

Comment:
Is the value stored in this variable every used? I don't think it is.

Reply:
It is; see lines 43–87 of this function

Reply:
Ah, sorry. I think when I was reviewing and left this comment I had mistakenly read this function as ending at line 41, rather than continuing past that long comment.